src/Setup/res/License.rtf

```diff
@@ -1,40 +1,40 @@
-\b0 15.\tab\b Governing Law/Jurisdiction:\b0  This Agreement shall be deemed to be made in, and the validity, interpretation and enforcement of this Agreement, and any disputes between you and NSB, shall be governed by and construed in accordance with the laws of, the State of New York, NY, excluding its conflicts of laws principles and excluding the United Nations Convention on Contracts for the International Sale of Goods. All disputes arising out of or relating to this Agreement shall be submitted to the exclusive jurisdiction of the courts of competent jurisdiction located in the state of New York, NY, and each party irrevocably consents to such personal jurisdiction and waives all objections to such venue. Notwithstanding the foregoing, we may seek injunctive relief in any court of competent jurisdiction worldwide. \line\par
-16.\tab\b Miscellaneous:\b0  This Agreement represents the complete agreement concerning the Software between NSB and Licensee and supersedes all prior agreements and representations between NSB and Licensee. If any provision of this Agreement is held to be unenforceable for any reason, such provision shall be reformed only to the extent necessary to make it enforceable. Any waiver of any provision of this Agreement will be effective only if in writing and signed by NSB. The terms of this Agreement are personal to Licensee and may not be assigned or transferred for any reason whatsoever without the consent of NSB and any action or conduct in violation of the foregoing shall be void and without effect. NSB may assign this Agreement and delegate any of its obligations hereunder without re\lang3081 striction. This Section \ltrmark 16 and Sections \ltrmark 4 (License Restrictions), \ltrmark 5 (Ownership Rights), \ltrmark 7 (Non-Disclosure), \ltrmark 8 (Feedback), \ltrmark 10 (Third Party Software), \ltrmark 11 (Exclusion of Warranty), \ltrmark 12 (Limited Liability), \ltrmark 13 (Government Use), and \ltrmark 15 (Governing Law an\lang1033 d Jurisdiction), and any provision so intended, and any outstanding obligations and rights previously accrued, shall survive termination or expiration of this Agreement.\b\par
-
-\pard\widctlpar\sb240\sa240\b0 YOU EXPRESSLY ACKNOWLEDGE THAT YOU HAVE READ THESE TERMS AND UNDERSTAND THE RIGHTS, OBLIGATIONS, TERMS AND CONDITIONS SET FORTH HEREIN. BY CLICKING ON THE AGREE BUTTON AND/OR CONTINUING TO INSTALL OR USE THE SOFTWARE, YOU EXPRESSLY CONSENT TO BE BOUND BY THESE TERMS.\par
-}
+{\rtf1\ansi\ansicpg1252\deff0\nouicompat\deflang3081\deflangfe3081{\fonttbl{\f0\fswiss\fprq2\fcharset0 Calibri;}{\f1\froman\fprq2\fcharset0 Times New Roman;}{\f2\fswiss\fprq2 Calibri;}}
```

Comment:
THe file before and after is identical, I think that this checkin sneaked upon me ;)